src/axom/inlet/docs/sphinx/quick_start.rst

```diff
@@ -0,0 +1,124 @@
+.. _inlet_quick_start_label:
+
+Quick Start
+===========
+
+Inlet's workflow is broken into the four following steps:  
+
+* Defining the schema of your input deck
+* Verifying that the user given input deck is valid
+* Accessing the read and stored data in your program
+* *Optional* Generating documentation based on defined schema
+
+.. _inlet_defining_schema_label:
+
+Defining Schema
+---------------
+
+The first step in using Inlet is to define the schema of your input deck.
+Inlet defines an input deck into two basic classes: Tables and Fields. Basically
+Fields are individual values and Tables hold groups of Fields and Tables.
+
+Define the schema by using the following functions, on either the main Inlet class, for
+global Tables and Fields, or on individual Table classes, for Tables and Fields under that Table:
+
+========================= ===================
+Name                      Description
+========================= ===================
+addTable                  Adds a Table to the input deck schema with the given name.
+addBool                   Adds a boolean Field to the global or parent Table with the given name.
+addDouble                 Adds a double Field to the global or parent Table with the given name.
+addInt                    Adds a integer Field to the global or parent Table with the given name.
+addString                 Adds a string Field to the global or parent Table with the given name.
+========================= ===================
+
+All possible Tables and Fields that are can be found in the input deck must be defined
+at this step.  The value of the Field is read and stored into the Sidre datastore when you call the appropriate
+add function. Use the ``required`` class member function on the Table and Field class to indicate that
+they have to present in the given input deck. You can also set a default value to each field via the type-safe
+``Field::defaultValue()`` member functions. Doing so will populate the corresponding Fields value
+if the specific Field is not present in the input deck. The following example shows these concepts:
+
+.. literalinclude:: ../../examples/verification.cpp
+   :start-after: _inlet_workflow_defining_schema_start
+   :end-before: _inlet_workflow_defining_schema_end
+   :language: C++
+
+
+.. _inlet_verification_label:
+
+Verification
+------------
+
+This step helps ensure that the given input deck follows the rules expected by the code.  These
+rules are not verified until you call ``Inlet::verify()``.  Doing so will return true/false and
```

Comment:
Clarification: (I think it says this, but we might want to be explicit)
The `verify` happens after all data has been parsed, so the verification rules can access any other part of the input, right?